docs/docs/tutorials/creating-fake-user-data.md

```diff
@@ -0,0 +1,257 @@
+
+Integer columns are:
+* Sessions
+* Transactions 
+
+Floating type columns are:
+* Bounce rate 
+* Avg. Session Duration
+* Revenue
+* Goal Conversion Rate
+
+For all columns, we need numbers that are equal or greater than zero. Besides, for the percentage columns, we need to introduce the upper limit. The duration column will be expressed in minutes since it is a bit more intuitive than hours and seconds. Additionally, we need to specify a reasonable maximum duration as well. 
+
+---
+**note**
+`revenue` and `transactions` have a dependency: a revenue can only be above zero if at least one transaction took place. \
+You will need to filter your data later and generate additional rows if not enough data has been left.
+
+---
+
+The same data in JSON format:
+
+```json
+{
+    "users":[
+        {
+        "user id": "fcnMA33",
+        "sessions": 64,
+        "avg session duration": 6.71,
+        "bounce rate": 1.56,
+        "revenue": 0.00,
+        "transactions": 0,
+        "goal conversion rate": 96.886
+        },
+        {
+        "user id": "LAhfn66",
+        "sessions": 48,
+        "avg session duration": 7.99,
+        "bounce rate": 2.08,
+        "revenue": 0.00,
+        "transactions": 0,
+        "goal conversion rate": 33.33
+        }
+    ]
+}
+```
+
+In the `synth` [number generators reference](https://www.getsynth.com/docs/content/number), you will find out that the tool offers one generator for both types of numbers but the output can be customized using different arguments and parameters.
+
+
+## Creating Namespace and Collection
+
+Similarly to inserting data into a database, we need to create a "bucket" first:
+* a [namespace](https://www.getsynth.com/docs/getting_started/core-concepts#namespaces) where you will store your data [schemas](https://www.getsynth.com/docs/getting_started/schema): a new directory on your storage drive
+* a [collection](https://www.getsynth.com/docs/getting_started/core-concepts#collections) for each schema: a JSON file
+
+In your CLI tool, execute the following commands:
+
+```commandline
+$ mkdir fake_ga
+```
+```commandline
+$ cd fake_ga && touch users.json
+```
+
+Now, we can start coding the data model.
+
+## Coding Your Data Model
+
+You need to open the `users.json` file in a code editor and reproduce the data model.
+
+```json
+{
+    "type": "array",
+    "length": 2,
+    "content": {
+        "type": "object",
+        "user id": {},
+        "sessions": {},
+        "avg session duration": {},
+        "bounce rate": {},
+        "revenue": {},
+        "transactions": {},
+        "goal conversion rate": {}
+    }
+}
+```
+Then, let's fill in the fields with their generators.
+
+### User ID: A String Generator With Pattern
+
+We will use [this generator](https://www.getsynth.com/docs/content/string#pattern) to create user IDs. We can make them look GA-authentic by specifying a regex pattern:
```

Comment:
You should not use the domain in those links (e.g. `/docs/content/string#pattern`)